nix/modules/chainweb-node.nix

```diff
@@ -103,7 +97,13 @@ in
         peer-api = "server localhost:1789;";
       };
       extraHttpConfig = lib.optionalString cfg.throttle ''
-        limit_req_zone $binary_remote_addr zone=cwn:10m rate=10r/s;
+        limit_req_zone $binary_remote_addr zone=cwn:10m rate=20r/s;
+        limit_req_status 429;
+        limit_conn_status 429;
+        map $status $retry_after {
+            default ''';
```

Comment:
No, this is actually how you escape `''` inside a Nix multi-line string, since `''` is how you delimit Nix multi-line strings. `'''` escapes `''` and `''${` escapes `${`